ui/src/components/Issuer/IssuedToken.tsx

```diff
@@ -150,4 +152,17 @@ const IssuedToken: React.FC<Props> = ({ sideNav, onLogout }) => {
     }
 }
 
+
+const AllocationsChart = (props: { nettedTokenDeposits: DepositInfo[] }) => {
+    return (
+        <div className='allocations'>
+            <DonutChart data={formatNetTokenDeposits(props.nettedTokenDeposits)}/>
+        </div>
+    )
+
+    function formatNetTokenDeposits(tokens: DepositInfo[]): IDonutChartData[] {
+        return tokens.map(t => {return {title: `${t.investor}/${t.provider}`, value: t.quantity, color: getDonutChartColor(tokens.indexOf(t))}})
```

Comment:
suggestion: adding some new lines in here will help make this more readable, e.g. 

```suggestion
        return tokens.map(t => {
            return {
                title: `${t.investor}/${t.provider}`, 
                value: t.quantity, 
                color: getDonutChartColor(tokens.indexOf(t))
            }
        })
```